README.md

```diff
@@ -1 +1,43 @@
-# guidebook-googledrive-integration
\ No newline at end of file
+# About
+This project contains code for syncing Google Drive data with a Guidebook Guide.
+
+# Setup
+### What you will need
+1. An [Amazon Web Services (AWS) account](https://aws.amazon.com/)
+2. A Guide set up in [Builder](https://builder.guidebook.com/) with an empty CustomList
+3. An API key from Builder
+4. A Google Drive folder
+5. A Google Cloud Platform project
+
+## Environment Setup
+1. Create a [virtualenv](https://virtualenv.pypa.io/en/stable/) and run `pip install -r requirements.txt` to get the package dependencies.
+
+## Initial Data Load
+This project uses a scheduled Google Drive lambda to add, update and remove files as CustomListItems in Builder. However, an initial data load must be performed to populate the guide with data.
+
+**Steps to load data:**
+1. Update all of the values in `settings.py` to your settings
+2. Run `python data_loader.py` from within your virtualenv.
+
+## Google Drive Setup
+1. Create a Google Cloud Platform project
+2. In IAM and Admin for this project, create a service account. This will be used for authentication.
```

Comment:
might be nice to link to this article https://cloud.google.com/iam/docs/creating-managing-service-accounts